__tests__/app.test.js

```diff
@@ -1,39 +1,102 @@
-	test("404 status code and error message if path not found", () => {
+describe('path not found', () => {
+	test('404 status code and error message if path not found', () => {
 		return request(app)
-			.get("/api/not-a-path")
+			.get('/api/not-a-path')
 			.expect(404)
 			.then((response) => {
-				expect(response.body.message).toBe("path not found");
-			});
-	});
-});
+				expect(response.body.message).toBe('path not found')
+			})
+	})
+})
 
-describe("/api/topics", () => {
-	test("GET:200 responds with an array of topics", () => {
+//articles
+describe('/api/articles', () => {
+	test('responds with 200 status code', () => {
+		return request(app).get('/api/articles/1').expect(200)
+	})
+	test('GET:200 responds with an array of article objects', () => {
 		return request(app)
-			.get("/api/topics")
+			.get('/api/articles')
 			.expect(200)
 			.then((response) => {
-				expect(response.body.topics.length).toBe(3);
-				response.body.topics.forEach((topic) => {
-					expect(typeof topic.slug).toBe("string");
-					expect(typeof topic.description).toBe("string");
-				});
-			});
-	});
-});
+				return console.log(response.body.articles)
+				expect(response.body.articles.length).toBe(data.articleData.length)
+				response.body.articles.forEach((article) => {
+					expect(typeof article.title).toBe('string')
+					expect(typeof article.topic).toBe('string')
+					expect(typeof article.author).toBe('string')
+					expect(typeof article.body).toBe('string')
+					expect(typeof article.created_at).toBe('string')
+					expect(typeof article.votes).toBe('number')
+					expect(typeof article.article_img_url).toBe('string')
+				})
+			})
+	})
+
+	test('GET:200 responds with an array of article objects sorted by date in descending order.', () => {
+		return request(app)
+			.get('/api/articles')
+			.then((response) => {
+				const { articles } = response.body
+
+				const sortedArticles = [...articles].sort(
+					// convert date string back into number (ms since epoch)
+					(a1, a2) => Date.parse(a1.created_at) - Date.parse(a2.created_at)
+				)
+
+				console.log(sortedArticles)
+			})
+	})
+})
+
+describe('GET /api/articles/:article_id', () => {
+	test('responds with 200 status code', () => {
+		return request(app).get('/api/articles/1').expect(200)
+	})
+	test('GET:200 sends a single article to the client', () => {
+		return request(app)
+			.get('/api/articles/1')
+			.then(({ body }) => {
+				expect(body.article.title).toBe('Living in the shadow of a great man')
+				expect(body.article.topic).toBe('mitch')
+				expect(body.article.author).toBe('butter_bridge')
+				expect(body.article.body).toBe('I find this existence challenging')
+				expect(body.article.created_at).toBe('2020-07-09T20:11:00.000Z')
+				expect(body.article.votes).toBe(100)
+				expect(body.article.article_img_url).toBe(
+					'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+				)
```

Comment:
One thing to try when you're looking at the structure of an object in jest tests is toMatchObject [https://jestjs.io/docs/expect#tomatchobjectobject]. Read the docs for it and let me know what you think but it's a nicer syntax and does the same job as what you've written.

Reply:
You should definitely be checking that the article ID is 1 in the returned object too